jest.json

```diff
@@ -4,8 +4,8 @@
   "collectCoverageFrom": [
     "packages/**/src/**/*.ts",
     "!**/examples/**",
```

Comment:
The examples folder is also gone, correct?

Reply:
Correct.

We'll also rename a bunch of example projects and point to them, but that will be done later.